lib/Service/DefaultBoardService.php

```diff
@@ -71,19 +79,62 @@ public function checkFirstRun($userId): bool {
 	 * @throws BadRequestException
 	 */
 	public function createDefaultBoard(string $title, string $userId, string $color) {
+		/** @var Board $defaultBoard */
 		$defaultBoard = $this->boardService->create($title, $userId, $color);
 		$defaultStacks = [];
 		$defaultCards = [];
 
 		$boardId = $defaultBoard->getId();
 
-		$defaultStacks[] = $this->stackService->create($this->l10n->t('To do'), $boardId, 1);
-		$defaultStacks[] = $this->stackService->create($this->l10n->t('Doing'), $boardId, 1);
-		$defaultStacks[] = $this->stackService->create($this->l10n->t('Done'), $boardId, 1);
+		$defaultStacks[] = $this->stackService->create(
+			$this->l10n->t('Custom lists - click to rename!'),
+			$boardId,
+			0
+		);
+		$defaultStacks[] = $this->stackService->create($this->l10n->t('Todo'), $boardId, 1);
+		$defaultStacks[] = $this->stackService->create($this->l10n->t('In Progress'), $boardId, 2);
+		$defaultStacks[] = $this->stackService->create($this->l10n->t('Done'), $boardId, 3);
 
-		$defaultCards[] = $this->cardService->create($this->l10n->t('Example Task 3'), $defaultStacks[0]->getId(), 'text', 0, $userId);
-		$defaultCards[] = $this->cardService->create($this->l10n->t('Example Task 2'), $defaultStacks[1]->getId(), 'text', 0, $userId);
-		$defaultCards[] = $this->cardService->create($this->l10n->t('Example Task 1'), $defaultStacks[2]->getId(), 'text', 0, $userId);
+		$readMoreLabel = $this->labelService->create('Read more inside', 'CC317C', $boardId);
+		$defaultLabels = $defaultBoard->getLabels();
+
+		$actionLabel = null;
+		foreach ($defaultLabels as $label) {
+			if ($label->getTitle() === 'Action needed') {
+				$actionLabel = $label;
+				break;
+			}
+		}
+
+		$cardsDataJson = file_get_contents(__DIR__ . '/fixtures/default-cards.json');
+		$cardsData = json_decode($cardsDataJson, true);
+
+		foreach ($cardsData as $cardData) {
+			$stackIndex = $cardData['stackIndex'];
+			$defaultCards[] = $this->cardService->create(
+				$this->l10n->t($cardData['title']),
```

Comment:
This won't work. The translations are extracted and then put for translation based on regex matches for the t() method calls, so the strings in the json file would not be found by that.

We should rather have the card title strings defined in PHP directly using the t method calls.